sgkit/stats/popgen.py

```diff
@@ -406,7 +406,7 @@ def Tajimas_D(
     S = ((ac > 0).sum(axis=1) > 1).sum()
 
     # assume number of chromosomes sampled is constant for all variants
-    n = ac.sum(axis=1).max()
+    n = ac.sum(axis=1).max().compute()
```

Comment:
I'm not sure if there's a way to make `Tajimas_D`, so this `compute` is needed for the time being at least. This was effectively being hidden by `variant_allele_count` not being lazy.

Reply:
@tomwhite we should probably open an issue to remove this compute, and make it lazy again?

Reply:
FYI https://github.com/pystatgen/sgkit/issues/354